metrics/vaultmetrics.go

```diff
@@ -0,0 +1,37 @@
+package metrics
+
+import (
+	"github.com/ecadlabs/signatory/signatory"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var vaultSigningSummary = prometheus.NewSummaryVec(
+	prometheus.SummaryOpts{
+		Name: "vault_sign_request_durations",
```

Comment:
Rename `vault_sign_request_durations` to `vault_sign_request_duration_seconds` to align with Prometheus metric naming guidelines; https://prometheus.io/docs/practices/naming/